include/fox/inplace_free_list.hpp

```diff
@@ -98,6 +98,11 @@ namespace fox
 			return size_;
 		}
 
+		[[nodiscard]] offset_type first_free_offset() const noexcept
```

Comment:
Make this private, `_first_free_offset`. Exposing this doesn't provide any benefits for the library user.